daprdocs/content/en/reference/resource-specs/component-schema.md

```diff
@@ -8,27 +8,31 @@ description: "The basic spec for a Dapr component"
 
 Dapr defines and registers components using a [resource specifications](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/). All components are defined as a resource and can be applied to any hosting environment where Dapr is running, not just Kubernetes.
 
+{{% alert title="Note" color="primary" %}}
+Any component can be restricted to a particular [namepsace]({{< ref isolation-concept.md >}}) and restricted access through scopes to any particular set of applications.
+{{% /alert %}}
```

Comment:
I think it is worth noting that a Component is _always_ restricted to a particular namespace- there is no concept of a "global" Component which can be ingested by all daprds regardless of namespace. This namespace is either explicit on the Component manifest itself, or set by the API server which derives the namespace through context when applying to Kubernetes. The only exception to this rule is in self-hosted mode whereby daprd will ingest Component (or other) resources when the namespace field is omitted, though the security profile of this is mute as daprd has access to the manifest anyway, unlike in kube.

Reply:
@JoshVanL thanks! I tried to clarify this in the doc, if you could take a look?

Reply:
LGTM!